.travis.yml

```diff
@@ -1,11 +1,14 @@
-dist: xenial
+dist: trusty
 
 language: java
 
 jdk: oraclejdk8
 
 sudo: required
```

Comment:
do we still need sudo at this point?

Reply:
Hmm, I believe we do. It makes Travis to run inside VM instead of a container. I thought you needed in order to run docker. 

I can double check what happens if I run with sudo required = False